src/core/api/__tests__/public_api.test.ts

```diff
@@ -0,0 +1,561 @@
+      });
+    });
+
+    describe("getPlaybackRate", () => {
+      it("should return 1 in getPlaybackRate by default", () => {
+        expect(player.getPlaybackRate()).to.equal(1);
+      });
+    });
+
+    describe("getVolume", () => {
+      it("should return 1 in getVolume by default", () => {
+        expect(player.getVolume()).to.equal(1);
+      });
+    });
+
+    describe("isFullscreen", () => {
+      it("should return false in isFullscreen by default", () => {
+        expect(player.isFullscreen()).to.equal(false);
+      });
+    });
+
+    describe("getAvailableVideoBitrates", () => {
+      it("should return [] in getAvailableVideoBitrates by default", () => {
+        expect(player.getAvailableVideoBitrates()).to.eql([]);
+      });
+    });
+
+    describe("getAvailableAudioBitrates", () => {
+      it("should return [] in getAvailableAudioBitrates by default", () => {
+        expect(player.getAvailableAudioBitrates()).to.eql([]);
+      });
+    });
+
+    describe("getVideoBitrate", () => {
+      it("should return undefined in getVideoBitrate by default", () => {
+        expect(player.getVideoBitrate()).to.equal(undefined);
+      });
+    });
+
+    describe("getAudioBitrate", () => {
+      it("should return undefined in getAudioBitrate by default", () => {
+        expect(player.getVideoBitrate()).to.equal(undefined);
+      });
+    });
+
+    describe("getMaxVideoBitrate", () => {
+      it("should return Infinity in getMaxVideoBitrate by default", () => {
+        expect(player.getMaxVideoBitrate()).to.equal(Infinity);
+      });
+    });
+
+    describe("getMaxAudioBitrate", () => {
+      it("should return Infinity in getMaxAudioBitrate by default", () => {
+        expect(player.getMaxAudioBitrate()).to.equal(Infinity);
+      });
+    });
+
+    describe("getWantedBufferAhead", () => {
+      it("should return 30 in getWantedBufferAhead by default", () => {
+        expect(player.getWantedBufferAhead()).to.equal(30);
+      });
+    });
+
+    describe("getMaxBufferBehind", () => {
+      it("should return Infinity in getMaxBufferBehind by default", () => {
+        expect(player.getMaxBufferBehind()).to.equal(Infinity);
+      });
+    });
+
+    describe("getMaxBufferAhead", () => {
+      it("should return Infinity in getMaxBufferAhead by default", () => {
+        expect(player.getMaxBufferAhead()).to.equal(Infinity);
+      });
+    });
+
+    describe("getPlaybackRate/setPlaybackRate", () => {
+      it("should allow to change the playback rate through setPlaybackRate", () => {
+        expect(player.setPlaybackRate(4)).to.equal(undefined);
+        expect(player.getPlaybackRate()).to.equal(4);
+
+        expect(player.setPlaybackRate(3)).to.equal(undefined);
+        expect(player.getPlaybackRate()).to.equal(3);
+
+        expect(player.setPlaybackRate(2)).to.equal(undefined);
+        expect(player.getPlaybackRate()).to.equal(2);
+
+        expect(player.setPlaybackRate(1.5)).to.equal(undefined);
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Remove this use of the output from "setPlaybackRate"; "setPlaybackRate" doesn't return anything. ([typescript:S3699](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS3699))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=436&issues=AWgOFkKwUMkE2J58zqec&open=AWgOFkKwUMkE2J58zqec)

[ISSUE_KEY]: AWgOFkKwUMkE2J58zqec